File: Group 85-b/level 39/homework/hw3.py
##4. იგივე თაფლის გამოყენებით გამოიტანეთ უდიდესი და უმცირესი რიცხვი (max() / min() ფუნქციების გარეშე)
tuple = (1, 2, 3, -4, 50, -6, 89, -100, 700, -3, 0.5, 0)
max_num = tuple[0]
min_num = tuple[0]
for num in tuple:
    if num > max_num:
        max_num = num
    if num < min_num:
        min_num = num
print("Maximum number:", max_num)
print("Minimum number:", min_num)

File: Group 85-b/level 55/homework/hw3.js
//დაწერეთ კოდი, რომელიც აიღებს მასივს [3,6,9,12], ყველა რიცხვის ჯამი გამოთვალოს
let number = [3,6,9,12];
let sum = 0;
for (let i = 0; i <number.length; i++) {
    sum += number[i];
}
console.log(sum);